src/main/java/com/sahabuddin/blogappapis/payloads/SignInRequest.java

```diff
@@ -0,0 +1,18 @@
+package com.sahabuddin.blogappapis.payloads;
+
+import jakarta.validation.constraints.NotEmpty;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class SignInRequest {
+
+    @NotEmpty
```

Comment:
Use custom message with validation annotations and handle the validation exception properly so that the end user can get the appropriate validation fail message.